notebooks/swarm.ipynb

```diff
@@ -0,0 +1,1106 @@
+{
```

Comment:
<p>I didn't understand exactly why we introduce a ToolCallingAgent here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
This is not a complete `SwarmAgent` because it cannot do handoffs.
Let's discuss about this. Maybe I can explain it better...

Reply:
<p>I will better explain the difference in the Handoff section.</p>